__tests__/app.test.js

```diff
@@ -114,3 +114,41 @@ describe("GET /api/articles", () => {
         })
       })
 })
+
+describe("GET /api/articles/:article_id/comments", () => {
+    test("responds with an array of comments for the given article_id", () => {
+        return request(app)
+          .get("/api/articles/1/comments")
+          .expect(200)
+          .then(({ body }) => {
+            const { comments } = body
+            expect(Array.isArray(comments)).toBe(true);
```

Comment:
We should always be as specific as we can when testing, currently we are only asserting against the keys could we assert against the values as well?